docs/getting-started/user-guides/decrediton-setup.md

```diff
@@ -6,13 +6,13 @@ Last updated for v1.0.0.
 
 `Decrediton` is a graphical user interface for `dcrwallet`. When this application launches, it automatically starts its own instance of `dcrd` and `dcrwallet` in the background - it will not open if there is already a running instance of `dcrd`.
 
-NOTE: At any point in the use of Decrediton, it may become unresponsive or get stuck on a loading screen. This can usually be fixed with a restart of the application.
+NOTE: If at any point the program becomes unresponsive or gets stuck on a loading screen, this can usually be fixed with a restart of the application.
 
 ---
 
 ## **Download and Install** ##
 
-Decrediton is released with the Binary Releases and can be found here: [https://github.com/decred/decred-binaries/releases/tag/v1.0.5_decrediton](https://github.com/decred/decred-binaries/releases/tag/v1.0.5_decrediton). As of v1.0.5, Decrediton is only available for Linux and Mac.
+Decrediton is released with the Binary Releases and can be found here: [https://github.com/decred/decred-binaries/releases/tag/v1.0.5]. As of v1.0.5, Decrediton is only available for Linux and Mac.
```

Comment:
Now that doesn't show up as a link.  What I wanted was:
`[https://github.com/decred/decred-binaries/releases/tag/v1.0.5](https://github.com/decred/decred-binaries/releases/tag/v1.0.5)`